aybapi/base/client.py

```diff
@@ -0,0 +1,36 @@
+import abc
+import typing
+from ..impl.bot import Bot
```

Comment:
why are you sometimes using relative imports and sometimes using absolute imports?